### PR TITLE
Add tech blog post pt. 2

### PR DESCRIPTION
**Pull request by vestimir**
🥈

### CODE REVIEW

File: source/blog/2017-06-06-functional-technologies-on-the-web-pt2.html.markdown [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+- Processes are strongly isolated.
+- Process creation and destruction is a lightweight operation.
+- Message passing is the only way for processes to interact.
+- Processes have unique names.
+- If you know the name of a process you can send it a message.
+- Processes share no resources.
+- Error handling is non-local.
+- Processes do what they are supposed to do or fail.
+
+Or simply put: [“If Java is 'write once, run anywhere', then Erlang is 'write once, run forever'.”](https://www.youtube.com/watch?v=u41GEwIq2mE&t=3m59s)
```
Comment (gsamokovarov) [inline source/blog/2017-06-06-functional-technologies-on-the-web-pt2.html.markdown]: https://www.youtube.com/watch?v=uKfKtXYLG78

File: source/blog/2017-06-06-functional-technologies-on-the-web-pt2.html.markdown [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+```erlang
+-module(hello).
+-export([hello_world/0]).
+
+hello_world() -> io:fwrite("Hello, Partial\n").
+```
+
+*If you use Erlang for your projects, Partial is an opportunity to share your experience with others. [Apply](https://goo.gl/qGfmds) for an Erlang talk!*
+
+## Elixir
```
Comment (vestimir) [inline source/blog/2017-06-06-functional-technologies-on-the-web-pt2.html.markdown]: DONE

File: source/blog/2017-06-06-functional-technologies-on-the-web-pt2.html.markdown [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+IO.puts "Hello Partial!"
+```
+
+At Partial, we’ll be covering Elixir with speakers like Saša Jurić (author of “Elxir in action”) and Michał Muskała (core member of the Ecto team).
+
+*If you want to delve deeper and learn more about Elixir – be sure to [get your ticket for Partial](http://partialconf.com/tickets) today!*
+
+## [Elm](http://elm-lang.org)
+
+Developers familiar with React, Redux and other popular web UI technologies will appreciate the simplicity of Elm. It’s a safe and fun way of building interfaces in the browser. The core principle is
```
Comment (svetliomihailov) [inline source/blog/2017-06-06-functional-technologies-on-the-web-pt2.html.markdown]: This seems like an unfinished sentence.

**Comment by svetliomihailov** [comment]
looks good to me.

**Pull request merged**